BLaurenB/techtime
# Retired Freelancers cannot be added to cart: update index page

#### What's this PR do?
Removes "choose me" button for retired freelancers on the freelancer index.
Adds 1 test to cover this scenario.

#### Where should the reviewer start- what are the pertinent changes and the sequence?
spec/features/freelancers/visitor_visits_freelancer_index_spec.rb:19 - "I see retired freelancers and I can't add them to the cart"
app/views/freelancers/index.html.erb - see the conditional to remove the choose me button if a freelancer is retired

#### How should this be manually tested?
start up server and visit the freelancers index. there should be 3 freelancers who are listed as retired and don't have a button.

#### Any background context you want to provide?
We already had the functionality for an individual freelancer's page, so this PR just takes that functionality and updates it for the index page.

#### What are the relevant tickets?
Close #123 

#### Screenshots (if appropriate)
#### Questions:
